cmd/notary/main.go

```diff
@@ -1,96 +1,71 @@
 package main
 
 import (
-	"crypto/x509"
 	"fmt"
 	"os"
 	"os/user"
-	"path"
-	"time"
+	"path/filepath"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 
-	"github.com/docker/notary/trustmanager"
+	"github.com/docker/notary/pkg/passphrase"
 )
 
 const configFileName string = "config"
-const configPath string = ".docker/trust/"
-const trustDir string = "trusted_certificates/"
-const privDir string = "private/"
-const rootKeysDir string = "root_keys/"
+const defaultTrustDir string = ".notary/"
+const defaultServerURL = "https://notary-server:4443"
```

Comment:
This is kind of weird too. notary-server won't exist unless someone adds it to a host file. Maybe it's better to have no default, and require the user to specify the server on the command line or in the config file. The error message would be more helpful than a message about not being able to resolve notary-server.